### ISSUE TITLE
Can't reach reported accuracy on Omniglot

### ISSUE THREAD

**Issue by xingzhaolee** [wyharveychen/CloserLookFewShot#14]
Hi, I've run the baseline Conv4 on Omniglot. I've notice that for 1-shot, the experiment result is far off from the reported 94%. Any idea why?

```
Time: 20190416-134341, Setting: omniglot-novel-Conv4S-baseline 5shot 5way_test, Acc: 600 Test Acc = 98.62% +- 0.15%
Time: 20190416-133451, Setting: omniglot-novel-Conv4S-baseline 1shot 5way_test, Acc: 600 Test Acc = 89.30% +- 0.58%
```

The scripts used to run these experiments are:
```bash
python3 train.py --dataset omniglot --model Conv4 --method baseline --num_classes 4112
python3 save_features.py --dataset omniglot --model Conv4 --method baseline
python3 test.py --dataset omniglot --model Conv4 --method baseline --n_shot 5
python3 test.py --dataset omniglot --model Conv4 --method baseline --n_shot 1
```

**Comment by wyharveychen** [comment]
Hello, since Omniglot does not use data augmentation, it is very likely to overfit.
I have performed validation and found that it would be better to stop within 5 epochs as stated in the appendix of the paper, thanks!

**Comment by xingzhaolee** [comment]
Sorry, do you mean stop within 5 epochs during training or finetuning one novel classes?

**Comment by wyharveychen** [comment]
Stop within 5 epochs during training (yes, very short training period)

**Comment by xingzhaolee** [comment]
Alright, I've got similar results.

5-shot: 99.14% +- 0.11%
1-shot: 94.68% +- 0.44%

This is the exact command I used in case someone else has this issue.
```
python3 train.py --dataset omniglot --model Conv4 --method baseline --num_classes 4112 --stop_epoch 5 --save_freq 5
python3 save_features.py --dataset omniglot --model Conv4 --method baseline --save_iter 4
python3 test.py --dataset omniglot --model Conv4 --method baseline --n_shot 5 --save_iter 4
python3 test.py --dataset omniglot --model Conv4 --method baseline --n_shot 1 --save_iter 4
```

**Comment by janetwise** [comment]
Got similar results using baseline and baseline++ on omniglot. However, only got 96% using protonet, compared with the 99% in the paper. Anyone got similar results? Tried epoch 5, 10... but all led to 96% test.

python3 train.py --dataset omniglot --model Conv4 --method protonet --num_classes 4112 --stop_epoch 5 --save_freq 5
